### PR TITLE
chore(web): merges master into feature-gestures, updates branch scripts

### PR DESCRIPTION
**Pull request by jahorton**
We've had quite a few updates to the builder-script structure since the feature branch was last updated, so it's high time to sync the feature-branch with all the latest changes.

https://github.com/keymanapp/keyman/pull/7432/files/5780da2cbf81d5ede84c38d2be6c2e31392ab994..b982afc5a96687b4233d5879844be7b2c9207b59 provides a view into all relevant post-merge updates.

@keymanapp-test-bot skip

### CODE REVIEW

File: common/web/gesture-recognizer/src/test/auto/headless/segmentConstruction.js [hunk 1]
```diff
@@ -527,7 +527,7 @@ describe("Iterative Segment Construction (ConstructingSegment)", function() {
         assert.isTrue(constructor.isCompatible(followup)); // To verify the following method's precondition.
         assert.isTrue(constructor.updatePendingSubsegment(followup));
 
-        followup = turtle.move(90, 100, 20, 20);
+        followup = turtle.move(90, 1, 20, 20);
```
Comment (jahorton) [inline common/web/gesture-recognizer/src/test/auto/headless/segmentConstruction.js:530]: The unit test's title indicated that this should have been a `1`, not a `100`:

```
 hold -> 2px move -> hold -> 1px move
```

The changed value reflects the number of pixels the simulated touch-move should travel.

**Review by mcdurdin — APPROVED** [review]
LGTM. Glad to see you fixed up the new builder scripts :grin: Sorry for the moving target!

File: common/web/gesture-recognizer/src/test/auto/headless/segmentConstruction.js (hunk 1 above)
Comment (mcdurdin) [reply common/web/gesture-recognizer/src/test/auto/headless/segmentConstruction.js:530]: Ha ha, was wondering about this, but figured you knew what you were doing ;-)

File: common/web/gesture-recognizer/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "@types/node": "^11.9.4",
     "@types/sinon": "^10.0.13",
     "chai": "^4.3.4",
-    "karma": "^6.3.4",
+    "karma": "^6.4.1",
```
Comment (jahorton) [inline common/web/gesture-recognizer/package.json:9]: Web updated this in its package.json; just a synchronization here.

**Pull request merged**